src/world/sound/BlastFurnaceFireCrackleSound.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\world\sound;
+
+use pocketmine\math\Vector3;
+use pocketmine\network\mcpe\protocol\LevelSoundEventPacket;
+use pocketmine\network\mcpe\protocol\types\LevelSoundEvent;
+
+final class BlastFurnaceFireCrackleSound implements Sound{
```

Comment:
Just because the Mojang names are absurdly inconsistent doesn't mean PM should be too. This can simply be named `BlastFurnaceLitSound` or even just `BlastFurnaceSound`. Same for the others.

Reply:
``BlastFurnaceSound`` may be clearer than ``BlastFurnaceLitSound``, the ``FurnaceLitSound`` should be renamed to ``FurnaceSound``. There is, for the moment, only one sound produced by the furnace, not requiring to specify the fact that they are active.